astropy/coordinates/builtin_frames/astrometric.py

```diff
@@ -124,9 +123,9 @@ def astrometric_to_astrometric(from_astrometric_coord, to_astrometric_frame):
         # Otherwise, the transform occurs just by setting the new origin.
         return to_astrometric_frame.realize_frame(from_astrometric_coord.cartesian)
```

Comment:
Is it possible / worth the effort to add a test covering this case?
https://coveralls.io/builds/6223606/source?filename=astropy%2Fcoordinates%2Fbuiltin_frames%2Fastrometric.py#L124